py/server/deephaven_internal/plugin/register.py

```diff
@@ -7,17 +7,19 @@
 
 from typing import Union, Type
 from deephaven.plugin import Plugin, Registration
-from deephaven.plugin.object import ObjectType
+from deephaven.plugin.object_type import ObjectType
```

Comment:
I didn't see `object_type` when I did a quick code scan.

Reply:
The PR description for https://github.com/deephaven/deephaven-plugin/pull/12 notes that we have not yet renamed this, since it makes reading the diff harder (git believes this is a remove+add, rather than a modify+rename).